web/packages/teleport/src/UnifiedResources/ResourceActionButton.tsx

```diff
@@ -271,7 +265,7 @@ const KubeConnect = ({ kube }: { kube: Kube }) => {
   );
 };
 
-const sortNodeLogins = (logins: string[]) => {
+export const sortNodeLogins = (logins: string[]) => {
```

Comment:
perhaps you already had this in mind, should we re-use this: https://github.com/gravitational/teleport/pull/34952/files#diff-b85276b287d12bb5c6ace6e14c78de19476484e30f16dee449546c7d2b7e0d04R55

Reply:
Yeah, this PR was made before the other one. In the end I'll make it so that we just share a single function.